pallets/author-mapping/src/lib.rs

```diff
@@ -200,10 +212,10 @@ pub mod pallet {
 		/// No new security deposit is required. Will replace `update_association` which is kept
 		/// now for backwards compatibility reasons.
 		#[pallet::weight(<T as Config>::WeightInfo::set_keys())]
-		pub fn set_keys(origin: OriginFor<T>, keys: KeysWrapper<T>) -> DispatchResult {
+		pub fn set_keys(origin: OriginFor<T>, keys: Vec<u8>) -> DispatchResult {
```

Comment:
if we make this change, we should just remove KeysWrapper

whole point of KeysWrapper was for the input

so I will just merge this branch into my branch and then do that refactor

Reply:
Yes, let me add an extra test first